popde/density_estimate.py

```diff
@@ -230,7 +230,8 @@ def calc_ndim_bandwidth(self, method='oned_isj'):
         if method == 'oned_isj':
             from KDEpy.bw_selection import improved_sheather_jones as isj
             for i, col in enumerate(self.kde_data.T):
-                nd_bws[i] = isj(col)  # Weighting is also possible, not implemented atm
+                #col should be a data of shape ndata, ndim==1
+                nd_bws[i] = isj(col.reshape(len(col), 1)) # Weighting is also possible, not implemented atm
```

Comment:
```suggestion
                nd_bws[i] = isj(col[:, np.newaxis]) # Weighting is also possible, not implemented atm
```
I think newaxis should be simpler